recipes/vcf-validator/meta.yaml

```diff
@@ -1,19 +1,23 @@
 {% set name = "vcf-validator" %}
-{% set version = "0.9.6" %}
+{% set version = "0.9.7" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_linux        # [linux]
-    sha256: 33fc99334e77a90e99415daebfa877a9bc0375a3d7c18809082ace64d8373cdd                                       # [linux]
-  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_assembly_checker_linux # [linux]
-    sha256: 7913a6b3f12fdf5a66c98026ef144be4f8964f981f80eca4b509e06e6b8282ab                                       # [linux]
-  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_macos        # [osx]
-    sha256: 68f50e38323260dcd77e236c27872e1a0eb5c6407736baf003cbc755c87bab94                                       # [osx]
-  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_assembly_checker_macos # [osx]
-    sha256: afb7383468fa08d597fc8f7588918dbcdafc8d6d41eb7d33d37ae71a30860cb7                                       # [osx]
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_linux               # [linux]
+    sha256: 49aef4841cd7d0913ba12020465830b834ccfe20f2094222c8c41511f6c801ac                                              # [linux]
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_assembly_checker_linux        # [linux]
+    sha256: a9a00303c05c1fb5a33a5bd1f5c74fb5d8097f92a1f3ee815e19c137c0e26f13                                              # [linux]
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_macos_x64           # [osx and x86]
+    sha256: 0b97a34fac6cf23951ffc33cdfb7b0602a89b4d5390564c8d816cb83019a3ffc                                              # [osx and x86]
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_assembly_checker_macos_x64    # [osx and x86]
+    sha256: 1b5404a4239da9f5e5e39bece975e4eea55d87c486121a076226829b79477bc3                                              # [osx and x86]
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_macos_arm64         # [osx and arm64]
```

Comment:
Support for OSX ARM64 is coming but not yet available!
See https://github.com/bioconda/bioconda-recipes/pull/46775

Reply:
Do you mean that the selector does not work ATM?

Reply:
The selector works but nothing will be build and uploaded.

Reply:
I see. Thank you for explaining.
Should I remove it from the build or leave it even if it does not get pulled ?

Reply:
As you prefer. It is just ignored at the moment, so it does not cause any harm.

Reply:
The `osx-arm64` builds have been enabled just now! You'll need to add the additional platform configuration: https://bioconda.github.io/developer/aarch64.html